### ISSUE TITLE
Users can upload an image

### ISSUE THREAD

**Issue by Tobitron** [MatthewRiggott/Best-Picture#15]
Rather than only letting people choose from FB photos, let them upload their own pictures.

**Comment by Tobitron** [comment]
@elliott-ribner do you mind doing this? I didn't do it in group projects and probly won't have time before I leave this weekend.

**Comment by ribner** [comment]
@Tobitron  Yes I think thats not a problem. I can do that with carrierwave.
